lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -250,26 +250,24 @@ <h3 class="hd hd-3">${_("Course Team Management")}</h3>
 
     <div class="auth-list-container"
       data-rolename="Group Moderator"
-      data-display-name="${_("Discussion Group Moderators")}"
+      data-display-name="${_("Group Community TA")}"
       data-info-text="
-        ${_("Discussion Group Moderators can edit or delete any post, clear misuse flags, close "
-         "and re-open threads, endorse responses, and see posts from all groups. "
-         "Their posts are marked as 'staff'. They cannot manage course team membership by "
-         "adding or removing discussion moderation roles. Only enrolled users can be "
-         "added as Discussion Moderators.")}"
+        ${_("Group Community TAs are members of the community who help course teams moderate discussions. Group "
+            "Community TAs see only posts by learners in their assigned group. They can edit or delete posts, "
+            "clear flags, close and re-open threads, and endorse responses, but only for posts by learners in "
+            "their group.")}"
```

Comment:
@staubina Sorry, I did not mean to eliminate the last 2 sentences when I made my review comment. After "only for posts by learners in their group." the same last 2 sentences as for Community TA should be added back: "Their posts are marked as 'Community TA'. Only enrolled learners can be added as Group Community TAs."
@sstack22 is this correct? We want their posts to be marked as "Community TA" and not as "Group Community TA"?